File: kiosco-pwa/src/api/gerencial.js
import client from './client'

const BASE = '/api/method/gcma_kiosco.api.gerencial'

export function getPanelGerencial360(fecha) {
  const params = {}
  if (fecha) params.fecha = fecha
  return client.get(`${BASE}.get_panel_gerencial_360`, { params })
}

export function getCoberturaMapa(fecha) {
  const params = {}
  if (fecha) params.fecha = fecha
  return client.get(`${BASE}.get_cobertura_mapa`, { params })
}

export function getReporteFotosCompetencia(limit = 50) {
  return client.get(`${BASE}.get_reporte_fotos_competencia`, { params: { limit } })
}

export function runAlertaAbandonoClientes(fecha) {
  const payload = {}
  if (fecha) payload.fecha = fecha
  return client.post(`${BASE}.run_alerta_abandono_clientes`, payload)
}

export function exportScorecardCsv(fecha) {
  const params = {}
  if (fecha) params.fecha = fecha
  return client.get(`${BASE}.export_scorecard_csv`, { params })
}

File: backend/gcma_kiosco/maroc_b2b/api/gerencial.py
"""Compatibilidad de namespace contractual: maroc_b2b.api.gerencial.*"""

from gcma_kiosco.api.gerencial import (
    export_scorecard_csv,
    get_cobertura_mapa,
    get_panel_gerencial_360,
    get_reporte_fotos_competencia,
    run_alerta_abandono_clientes,
)

__all__ = [
    "get_panel_gerencial_360",
    "get_cobertura_mapa",
    "get_reporte_fotos_competencia",
    "run_alerta_abandono_clientes",
    "export_scorecard_csv",
]
